public/video-ui/src/slices/s3Upload.ts

```diff
@@ -39,19 +39,24 @@ export const startVideoUpload = createAsyncThunk<
   unknown,
   { id: string; file: File; selfHost?: boolean }
 >('s3Upload/startVideoUpload', ({ id, file, selfHost }, { dispatch }) =>
-  createUpload(id, file, selfHost).then((upload: Upload) => {
-    dispatch(s3UploadStarted(upload));
+  createUpload(id, file, selfHost)
+    .then((upload: Upload) => {
+      dispatch(s3UploadStarted(upload));
 
-    const progress = (completed: number) =>
-      dispatch(s3UploadProgress(completed));
+      const progress = (completed: number) =>
+        dispatch(s3UploadProgress(completed));
 
-    return uploadParts(upload, upload.parts, file, progress)
-      .then(() => dispatch(setS3UploadStatusToComplete()))
-      .catch(err => {
-        dispatch(showError(errorDetails(err), err));
-        dispatch(setS3UploadStatusToError());
-      });
-  })
+      return uploadParts(upload, upload.parts, file, progress)
+        .then(() => dispatch(setS3UploadStatusToComplete()))
+        .catch(err => {
+          dispatch(showError(errorDetails(err), err));
+          dispatch(setS3UploadStatusToError());
+        });
+    })
+    .catch(err => {
```

Comment:
if we have this, can we remove the catch on the lines just above this? I think the promise rejection should fall through to here if not caught there, and we handle it in the same way

Reply:
thanks!